Sources/SwiftFormatRules/ValidateDocumentationComments.swift

```diff
@@ -153,4 +171,15 @@ extension Diagnostic.Message {
     "replace the singular inline form of 'Parameter' tag with a plural 'Parameters' tag "
       + "and group each parameter as a nested list"
   )
+
+  public static func removeThrowsComment(funcName: String) -> Diagnostic.Message {
+    return Diagnostic.Message(
+      .warning,
+      "remove the throws comment of \(funcName), it doesn't throw an error"
+    )
+  }
+
+  public static func documentErrorsThrown(funcName: String) -> Diagnostic.Message {
+    return Diagnostic.Message(.warning, "document the errors thrown by \(funcName)")
```

Comment:
Suggested message: `"add a 'Throws' tag describing the errors thrown by \(funcName)"`

Reply:
Fixed.